wpk/generate_wpk_package.sh

```diff
@@ -328,12 +341,22 @@ function main() {
             if [[ "${HAVE_PKG_NAME}" == true ]]; then
                 build_container ${WIN_BUILDER} ${WIN_BUILDER_DOCKERFILE} || clean ${WIN_BUILDER_DOCKERFILE} 1
                 local CONTAINER_NAME="${WIN_BUILDER}"
-                build_wpk_windows ${BRANCH} ${DESTINATION} ${CONTAINER_NAME} ${JOBS} ${PKG_NAME} ${OUT_NAME} ${CHECKSUM} ${CHECKSUMDIR} ${INSTALLATION_PATH} ${AWS_REGION} ${WPK_KEY} ${WPK_CERT} || clean ${WIN_BUILDER_DOCKERFILE} 1
+                build_wpk_windows_macos windows ${BRANCH} ${DESTINATION} ${CONTAINER_NAME} ${JOBS} ${PKG_NAME} ${OUT_NAME} ${CHECKSUM} ${CHECKSUMDIR} ${INSTALLATION_PATH} ${AWS_REGION} ${WPK_KEY} ${WPK_CERT} || clean ${WIN_BUILDER_DOCKERFILE} 1
```

Comment:
Change the function name and parameters. Set the target at the end `$13`.
The function could be named `pack_wpk`

Reply:
I changed the name of the function to `pack_wpk`, the parameter 'target' was no longer necessary when removing the duplicate option `pnm/pnw` (third comment)